src/modules/hip/kernel/crop_and_patch.hpp

```diff
@@ -0,0 +1,475 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+
+
+// -------------------- Set 0 - kernels for layout toggle (PKD3->PLN3, PLN3->PKD3) --------------------
+template <typename T>
+__global__ void convert_pkd3_pln3_hip_tensor(T *srcPtr,
```

Comment:
Is this the first gpu version of a simple pld3-pln3 toggle? 
This could be more useful as a helper in hip common if its generic.

Reply:
Yeah
It is a generic code. We did not place any HIP kernels in hip common before
So i thought i need to have in the code only

Will move this to hip common file